website/docs/d/ess_scaling_configurations.html.markdown

```diff
@@ -63,4 +63,6 @@ The following attributes are exported in addition to the arguments listed above:
     * `performance_level` - The performance level of the ESSD used as data disk.
   * `lifecycle_state` - Lifecycle state of the scaling configuration.
   * `creation_time` - Creation time of the scaling configuration.
+  * `instance_name` - InstanceName of an ECS instance.
```

Comment:
`instance_name` - (Optional,Available in 1.143.0+) InstanceName of an ECS instance.